includes/class-fontawesome.php

```diff
@@ -1363,8 +1371,52 @@ function( $html, $handle ) use ( $resource_collection ) {
 				);
 
 				if ( $load_spec['v4shim'] ) {
-					// phpcs:ignore WordPress.WP.EnqueuedResourceParameters
-					wp_enqueue_style( self::RESOURCE_HANDLE_V4SHIM, $resource_collection[1]->source(), null, null );
+					add_action(
+						'wp_enqueue_scripts',
+						function () use ( $resource_collection, $options, $license_subdomain, $version ) {
+						// phpcs:ignore WordPress.WP.EnqueuedResourceParameters
+							wp_enqueue_style( self::RESOURCE_HANDLE_V4SHIM, $resource_collection[1]->source(), null, null );
+
+							$font_face = <<< EOT
+@font-face {
+    font-family: "FontAwesome";
+    src: url("https://${license_subdomain}.fontawesome.com/releases/v${version}/webfonts/fa-brands-400.eot"),
+         url("https://${license_subdomain}.fontawesome.com/releases/v${version}/webfonts/fa-brands-400.eot?#iefix") format("embedded-opentype"),
+         url("https://${license_subdomain}.fontawesome.com/releases/v${version}/webfonts/fa-brands-400.woff2") format("woff2"),
+         url("https://${license_subdomain}.fontawesome.com/releases/v${version}/webfonts/fa-brands-400.woff") format("woff"),
+         url("https://${license_subdomain}.fontawesome.com/releases/v${version}/webfonts/fa-brands-400.ttf") format("truetype"),
+         url("https://${license_subdomain}.fontawesome.com/releases/v${version}/webfonts/fa-brands-400.svg#fontawesome") format("svg");
+}
+
```

Comment:
Here's the heart of the change, the @font-face definition.
1. accounting for all of the various formats, like our normal css does
2. with additional style overrides to account for those icons in v4 that map not to fas but to fab or far

Reply:
This looks good to me. We are just doing this to support pseudo elements correct? We probably need to test this in older IE versions to see where support falls apart so we can warn in the admin UI.